docs/mediapipe.md

```diff
@@ -0,0 +1,141 @@
+# Mediapipe {#ovms_docs_mediapipe}
+
+## Introduction
+MediaPipe is a production ready framework for building pipelines to perform inference over arbitrary sensory data. Using MediaPipe in the OVSM enables user to define a powerful graph from a lot of ready calculators/nodes that come with the MediaPipe which support all the needed features for running a stable graph like e.g. flow limiter node. User can also run the graph in a server or run it inside application host. Here can be found more information about [MediaPipe framework ](https://developers.google.com/mediapipe/framework/framework_concepts/overview)
```

Comment:
```suggestion
MediaPipe is an open-source framework for building pipelines to perform inference over arbitrary sensory data. Using MediaPipe in the OVMS enables user to define a powerful graph from a lot of ready calculators/nodes that come with the MediaPipe which support all the needed features for running a stable graph like e.g. flow limiter node. User can also run the graph in a server or run it inside application host. Here can be found more information about [MediaPipe framework ](https://developers.google.com/mediapipe/framework/framework_concepts/overview)
```

Reply:
Mediapipe is in 0.10 version so we shouldn't state that it is production ready. It's still in development.